### ISSUE TITLE
Move Ubuntu integration tests to build against jammy

### ISSUE THREAD

**Issue by nacc** [OSInside/kiwi#2128]
## Problem description

I was starting to explore using kiwi-ng for building Ubuntu images and the examples seem to use 21.04 (which is EOL) and get 404s for various paths mentioned. Additionally (and please CMIIW if it's just ignorance on my part) I did not see any repositories for Ubuntu at the URLS mentioned in the XML.

## Expected behaviour

All example build-tests work.

## Steps to reproduce the behaviour

Run build system on the Ubuntu example. I was trying the virtual profile.

## OS and Software information
+ KIWI version: Git
+ Operating system: Ubuntu 21.10
+ OBS version:

**Comment by nacc** [comment]
Ah, perhaps, I can use the versions from https://github.com/OSInside/kiwi-descriptions

**Comment by schaefi** [comment]
Hi, you are right the integration test for the ubuntu test images is based on 21.04. I assume you are referring to ```build-tests/x86/ubuntu/test-image-live-disk/appliance.kiwi```. I plan to move them to 22.04 (jammy) which was released recently.
For the ubuntu integration test images which builds in obs you are also right they cannot be easily build locally because all of ubuntu in the open build service is an on-demand import from some Ubuntu mirror. The kiwi built method to translate
obs repos like ```obs://Ubuntu:21.04/standard``` does not produce a valid url on the obs server.

I'm sorry that you had trouble here but to build them you would need to adapt the repo list manually

or:

> Ah, perhaps, I can use the versions from https://github.com/OSInside/kiwi-descriptions

yes this is where we maintain community image descriptions as examples and everybody is invited to add more of them :)
I recently added an example for jammy and these images were made to be build locally outside of obs

Does this help you to get started ?

Thanks
